installer/scripts/omsadmin.sh

```diff
@@ -862,6 +862,7 @@ make_dir()
     fi
 
     chown_omsagent $1
+    chmod 755 $1
```

Comment:
This chmod is expected to restrict permissions, to ensure that these folders do not have world-write access.

This change is meant to correct the permissions to how they were set prior to multi-homing changes: see this line in the old code: https://github.com/Microsoft/OMS-Agent-for-Linux/blob/f06b438b51665d4a4426f970af12c677f305a8ea/installer/datafiles/base_omsagent.data#L155